src/Handlers/FatalHandler.php

```diff
@@ -56,7 +52,7 @@ public function handle()
     protected function isFatal($lastError)
     {
         return
-            !is_null($lastError) &&
+            null !== $lastError &&
```

Comment:
This is most a style thing (PHP 8 optimizes this heavily), but across the code base `===` is more prevalent, so I'm slowly moving the code away from `is_null`:

```
❯ rg '(null\s*[!=]==|[!=]==\s*null)'  | wc -l
      68
❯ rg 'is_null' | wc -l
      26
```